src/components/member-card/card.module.scss

```diff
@@ -72,6 +72,21 @@
   width: 1rem;
 }
 
+.new_member_img_absent_container {
+  height: 80px;
+  width: 80px;
+  border-radius: 56px;
+  margin: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.new_member_img_absent_container_text {
```

Comment:
className can be better

Reply:
can u suggest a better classname?

Reply:
newMembersImg or newMembersIcon should work

Reply:
The current classname suggest that the image for the user is not present hence we are showing their initials, the suggested classname doesn't do that